Sources/spm-test-finder/main.swift

```diff
@@ -0,0 +1,48 @@
+#if os(OSX)
+
+import XCTest
+import Foundation
+
+enum Error: ErrorProtocol {
+    case invalidUsage
+    case unableToLoadBundle(String)
+}
+
+do {
+    guard Process.arguments.count == 2 else {
+        throw Error.invalidUsage
+    }
+    var bundlePath = Process.arguments[1]
+    if !(bundlePath as NSString).isAbsolutePath {
+        bundlePath = NSFileManager.default().currentDirectoryPath + "/" + bundlePath
+    }
+    bundlePath = (bundlePath as NSString).standardizingPath
+
+    guard let bundle = NSBundle(path: bundlePath) where bundle.load() else {
+        throw Error.unableToLoadBundle(bundlePath)
+    }
+    
+    let suite = XCTestSuite(forBundlePath: bundlePath)
```

Comment:
I think we should be using `XCTestSuite.defaultTestSuite()` here. I created a test bundle which links with a framework containing an `XCTestCase` subclass and observed that both Xcode and the `xctest` runner discover and run the tests from the framework. Likewise, `defaultTestSuite()` includes test cases found in linked frameworks, where as `XCTestSuite(forBundlePath:)` does not.